prntscraper/scraper.py

```diff
@@ -167,3 +177,19 @@ def folder_size(self):
 
         size = len(os.listdir(os.getcwd() + "/images"))
         return "\x1b[0;32;40m" f"There are currently {size} images in the folder."  "\x1b[0m"
+
+    def check_image(self, image_name):
```

Comment:
1. Should be changed to
```py
def __check_image()
```
It needs a double underscore so it can't be accessed outside the class.

2. Also can the function be moved below `__download_image()` as it's a helper function